apps/namadillo/src/App/Governance/GovernanceOverview.tsx

```diff
@@ -22,53 +23,55 @@ export const GovernanceOverview: React.FC = () => {
   // TODO: is there a better way than this to show that votedProposalIdsAtom
   // is dependent on isConnected?
   const extensionAtoms = isConnected ? [votedProposalIds] : [];
+  const activeAtoms = [allProposals, ...extensionAtoms];
 
-  useNotifyOnAtomError(
-    [allProposals, ...extensionAtoms],
-    [allProposals.isError, votedProposalIds.isError]
-  );
+  const liveProposals =
+    allProposals.data?.filter((proposal) => proposal.status === "ongoing") ||
+    [];
+
+  const upcomingProposals =
+    allProposals.data?.filter((proposal) => proposal.status === "pending") ||
+    [];
+
+  useNotifyOnAtomError(activeAtoms, [
```

Comment:
Should we still keep this toast now that the AtomErrorBoundary is in? I am happy either way.

Reply:
I think we can keep this. If we see that is too much, we can remove it at any time. I think it gives more emphasis to the error